micro-ui/web/micro-ui-internals/packages/css/CHANGELOG.md

```
@@ -0,0 +1,141 @@
+- Added styles for multi select dropdowns.
+
+## [1.8.0-beta.14]
+- Added tour help styles.
+
+## [1.8.0-beta.13]
+- Fixed the login text styling issue.
+
+## [1.8.0]
+- Workbench v1.0.
+
+## [1.8.0-beta.1]
```

Markdownlint

Expected: 1; Actual: 0; Below
Headings should be surrounded by blank lines

(MD022, blanks-around-headings)